include/cask/fiber/FiberImpl.hpp

```diff
@@ -19,6 +19,16 @@ namespace cask::fiber {
 
 enum FiberState { READY, RUNNING, WAITING, DELAYED, RACING, COMPLETED, CANCELED };
 
+#define PRINT_STATE(state) \
+    (state == READY ? "READY" : \
```

Comment:
A small nit:  It's good practice the wrap the use of the macro argument (i.e. 'state') with parenthesis inside the macro definition (.e. ((state) == READY ? "READY") in case you ever invoke the macro with an expression. I checked that all uses you have added don't use an expression, so you're definitely safe for current uses of the macro.

Reply:
The linter is also complaining about this, so will resolve.